usaspending_api/search/v2/elasticsearch_helper.py

```diff
@@ -0,0 +1,70 @@
+from django.conf import settings
+from elasticsearch import Elasticsearch
+from usaspending_api.awards.v2.lookups.elasticsearch_lookups \
+        import TRANSACTIONS_LOOKUP, award_type_mapping
+import logging
+logger = logging.getLogger('console')
+ES_HOSTNAME = settings.ES_HOSTNAME
+TRANSACTIONS_INDEX_ROOT = settings.TRANSACTIONS_INDEX_ROOT
+
+
+CLIENT = Elasticsearch(ES_HOSTNAME)
```

Comment:
This is OK for now. But it would be better to store this ES client create in a more centralized part of the codebase for any future endpoints to use. It could be a context manager to provide easier validation and error-messages